### PR TITLE
moving scripts to check servers/services here

### PR DESCRIPTION
**Pull request by javierwilson**
i'm moving this here as fabric seems to be a better tool for the job.
still at an early stage, it works by running:
$ fab icds check_status

### CODE REVIEW

File: fab/fabfile.py [hunk 1]
```diff
@@ -881,3 +891,16 @@ def reset_pillow(pillow):
     staticfiles.update_manifest,
     release.clean_releases,
 ]
+
+@task
+def check_status():
+    env.user = 'ansible'
+    env.sudo_user = 'root'
+    env.password = getpass('Enter the password for then ansbile user: ')
```
Comment (dannyroberts) [inline fab/fabfile.py]: then => the

**Review by dannyroberts — COMMENTED** [review]
Hey @javierwilson, for my own context, where are these scripts being moved from?

File: fab/fabfile.py (hunk 1 above)
Comment (javierwilson) [reply fab/fabfile.py]: good catch, i copy-and-pasted from L726 so fixing there too.

**Comment by javierwilson** [comment]
I started writing a ping.yml for ansible, to write a kind of "check_services" at the OS-level not HQ dependant.  Since ansible is not really equipped to do the job, I'm doing this here instead.

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**